--- ncarg-4.4.2/config/ymake.ppc64	2006-05-02 08:16:20.000000000 -0600
+++ ncarg-4.4.2/config/ymake	2007-06-27 10:28:13.000000000 -0600
@@ -364,19 +364,13 @@
         case    x*86:
         case    i*64:
         case    x*64:
-        case    p*64:
+        case    ppc*:
         case    alpha:
             set model   = $mach
             set arch    = $mach
             set sysincs = LINUX
             set vendor  = IBM
             breaksw
-        case    ppc:
-            set model   = $mach
-            set arch    = $mach
-            set sysincs = YDLINUX
-            set vendor  = APPLE
-            breaksw
         default:
             echo "$0 : Unknown machine type" > /dev/tty
             exit 1
