--- ncl_ncarg-6.0.0/ni/src/ncl/NclDataDefs.h	2011-05-27 12:53:25.000000000 -0600
+++ ncl_ncarg-6.1.0-beta/ni/src/ncl/NclDataDefs.h	2012-05-28 13:35:57.000000000 -0600
@@ -110,16 +110,24 @@
 NCL_obj = 	020000,
 NCL_list = 	040000,
 NCL_group = 	0100000,
-NCL_compound = 	0200000
+NCL_compound = 	0200000,
+NCL_opaque =	0400000,
+NCL_enum =	01000000
 } NclBasicDataTypes;
 
 typedef NclQuark string; /* Makes this a quark type */
 typedef char byte;
+typedef unsigned char ubyte;
+typedef unsigned short ushort;
+typedef unsigned int uint;
+typedef unsigned long ulong;
+typedef long long int64;
+typedef unsigned long long uint64;
 typedef int logical;
 typedef int obj;
 typedef int list;
 typedef int group;
-typedef int compound;
+typedef int nclcompound;
 
 typedef union _NclScalar {
 	double             doubleval;
@@ -140,7 +148,7 @@
 	logical            logicalval;
 	obj                objval;
 	group              groupval;
-	compound           compoundval;
+	nclcompound        compoundval;
 }NclScalar;
 
 typedef struct _NclRefList{
--- ncl_ncarg-6.0.0/ni/src/ncl/DataSupport.c.compound	2012-09-10 14:20:59.331255585 -0600
+++ ncl_ncarg-6.0.0/ni/src/ncl/DataSupport.c	2012-09-10 14:48:08.456212413 -0600
@@ -820,7 +820,7 @@
 		case NCL_group:
 			return(sizeof(group));	
 		case NCL_compound:
-			return(sizeof(compound));
+			return(sizeof(nclcompound));
 		case NCL_logical:
 			return(sizeof(int));	
 		case NCL_list:
