diff -up ncl_ncarg-5.2.1/config/Project.udunits ncl_ncarg-5.2.1/config/Project
--- ncl_ncarg-5.2.1/config/Project.udunits	2010-08-10 13:01:03.000000000 -0600
+++ ncl_ncarg-5.2.1/config/Project	2010-08-10 13:02:36.052660865 -0600
@@ -392,7 +392,7 @@ NGENV_DESCRIPT	= \
 		"graphcaps",NULL,"ncarg",NULL,NULL,		\
 		"colormaps",NULL,"ncarg",NULL,NULL,		\
 		"rangs","NCARG_RANGS","database",NULL,NULL,	\
-		"udunits","NCARG_UDUNITS","ncarg",NULL,NULL,	\
+		"udunits","UDUNITS_PATH",NULL,"/etc","udunits.dat",	\
 		"psadilookup","PSADILOOKUP_PATH",NULL,NULL,NULL,\
 		"examples",NULL,"ncarg",NULL,NULL,		\
 		"hluex",NULL,"ncarg",NULL,NULL,			\
