--- ncl_ncarg-5.0.0/config/Project.atlas	2007-12-14 07:59:49.000000000 -0700
+++ ncl_ncarg-5.0.0/config/Project	2007-12-14 08:02:47.000000000 -0700
@@ -402,8 +402,8 @@
 NIOLIB		= -lnio
 NCLLIB		=
 NCLAPILIB	= -lNGncl
-BLASLIB	        = -lblas_ncl
-LAPACKLIB       = -llapack_ncl
+BLASLIB	        = -lcblas -lf77blas -lptf77blas
+LAPACKLIB       = -llapack -latlas
 SPHERELIB	= -lsphere3.1_dp
 FFTPACK5LIB	= -lfftpack5_dp
 
@@ -446,8 +446,8 @@
 XCBLIB		= _UseLib(-L$(XCBLIBSRC),-lXcb)
 NCLLIB		= _UseLib(-L$(NCLLIBSRC),-lncl)
 NCLAPILIB	= _UseLib(-L$(NCLLIBSRC),-lnclapi)
-BLASLIB         = _UseLib(-L$(BLASSRC),-lblas_ncl)
-LAPACKLIB       = _UseLib(-L$(LAPACKSRC),-llapack_ncl)
+BLASLIB         = _UseLib(-L%LIBDIR%/atlas,-lcblas -lf77blas -lptf77blas)
+LAPACKLIB       = _UseLib(-L%LIBDIR%/atlas,-llapack -latlas)
 SPHERELIB	= _UseLib(-L$(SPHERESRC),-lsphere3.1_dp)
 FFTPACK5LIB	= _UseLib(-L$(FFTPACK5SRC),-lfftpack5_dp)
 
