--- ncl_ncarg-5.0.0/ncarview/src/lib/libncarg_ras/sgiraster.c.uint32	2000-08-22 09:12:18.000000000 -0600
+++ ncl_ncarg-5.0.0/ncarview/src/lib/libncarg_ras/sgiraster.c	2008-04-11 09:29:45.000000000 -0600
@@ -315,8 +315,8 @@
 		just to be safe.
 		*/
 
-		sgiinfo->tmpbuf = (UInt16_T *) ras_calloc(3*header->xsize, 1);
-		if (sgiinfo->tmpbuf == (UInt16_T *) NULL) {
+		sgiinfo->tmpbuf = (uint16_t *) ras_calloc(3*header->xsize, 1);
+		if (sgiinfo->tmpbuf == (uint16_t *) NULL) {
 			(void) ESprintf(errno, "malloc(%d)", (3*header->xsize));
 			return(RAS_ERROR);
 		}
--- ncl_ncarg-5.0.0/ncarview/src/lib/libncarg_ras/sgiraster.h.uint32	2000-08-21 21:30:25.000000000 -0600
+++ ncl_ncarg-5.0.0/ncarview/src/lib/libncarg_ras/sgiraster.h	2008-04-11 10:02:33.000000000 -0600
@@ -27,6 +27,7 @@
 #ifndef	__SGIRASTER_H__
 #define	__SGIRASTER_H__
 
+#include <stdint.h>
 
 #define SGI_FORMAT_NAME		"sgi"
 #define SGI_MAGIC 		0732
@@ -65,18 +66,6 @@
 is not (yet) that large.
 */
 
-#ifdef	alpha
-typedef	unsigned int	UInt32_T;
-typedef	unsigned short	UInt16_T;
-typedef	int		Int32_T;
-typedef	short		Int16_T;
-#else
-typedef	unsigned long	UInt32_T;
-typedef	unsigned short	UInt16_T;
-typedef	long		Int32_T;
-typedef	short		Int16_T;
-#endif
-
 #define RAS_SGI_RESERVED	512
 
 /*
@@ -84,15 +73,15 @@
 **
 */
 typedef	struct	{
-	UInt16_T	imagic B16;
-	UInt16_T 	type B16;
-	UInt16_T 	dim B16;
-	UInt16_T 	xsize B16;
-	UInt16_T 	ysize B16;
-	UInt16_T 	zsize B16;
-	UInt32_T 	min B32;
-	UInt32_T 	max B32;
-	UInt32_T	wastebytes B32;	
+	uint16_t	imagic B16;
+	uint16_t 	type B16;
+	uint16_t 	dim B16;
+	uint16_t 	xsize B16;
+	uint16_t 	ysize B16;
+	uint16_t 	zsize B16;
+	uint32_t 	min B32;
+	uint32_t 	max B32;
+	uint32_t	wastebytes B32;	
 	char 		name[80];
 	SgiColormapType	colormap B32;
 } SGIFileHeader_T;
--- ncl_ncarg-5.0.0/ncarview/src/lib/libncarg_ras/sunraster.h.uint32	2000-08-21 21:30:25.000000000 -0600
+++ ncl_ncarg-5.0.0/ncarview/src/lib/libncarg_ras/sunraster.h	2008-04-11 10:02:46.000000000 -0600
@@ -27,13 +27,9 @@
 #ifndef _RASTER_SUN_
 #define _RASTER_SUN_
 
-#define RAS_SUN_ESC	128
+#include <stdint.h>
 
-#ifdef	alpha
-typedef	unsigned int	UInt32_T;
-#else
-typedef	unsigned long	UInt32_T;
-#endif
+#define RAS_SUN_ESC	128
 
 /* Sun image encoding types. */
 
@@ -55,14 +51,14 @@
 #define SUN_HEADER_SIZE	32
 
 typedef struct SunInfoStruct {
-	UInt32_T	ras_magic B32;	/* magic number */
-	UInt32_T	ras_width B32;	/* width (pixels) of image */
-	UInt32_T	ras_height B32;	/* height (pixels) of image */
-	UInt32_T	ras_depth B32;	/* depth - 1,8,24 bits */
-	UInt32_T	ras_length B32;	/* length (bytes) of image */
+	uint32_t	ras_magic B32;	/* magic number */
+	uint32_t	ras_width B32;	/* width (pixels) of image */
+	uint32_t	ras_height B32;	/* height (pixels) of image */
+	uint32_t	ras_depth B32;	/* depth - 1,8,24 bits */
+	uint32_t	ras_length B32;	/* length (bytes) of image */
 	SunEncodingType	ras_type B32;
 	SunColormapType	ras_maptype B32;
-	UInt32_T	ras_maplength B32;/* length (bytes) of following map */
+	uint32_t	ras_maplength B32;/* length (bytes) of following map */
 } SunInfo;
 
 #define	RAS_MAGIC	0x59a66a95
