diff -up ncl_ncarg-5.2.0/ncarview/src/bin/ictrans/yMakefile.libs ncl_ncarg-5.2.0/ncarview/src/bin/ictrans/yMakefile
--- ncl_ncarg-5.2.0/ncarview/src/bin/ictrans/yMakefile.libs	1998-02-04 08:58:14.000000000 -0700
+++ ncl_ncarg-5.2.0/ncarview/src/bin/ictrans/yMakefile	2010-04-28 13:35:00.892361794 -0600
@@ -29,6 +29,8 @@ DEP_LIBS	=	$(DEPICTRANSLIB) $(DEPCTRANSL
 
 #ifdef  IRIX
 MORE_LIBS	= -lmalloc
+#else
+MORE_LIBS	= -lm
 #endif
 
 EXTRA_LIBS	= $(X11LIBS) $(SUNLIBS) $(NETCDFLIB) $(HDFLIB) $(CTOFLIBS)
diff -up ncl_ncarg-5.2.0/ni/src/ncl/yMakefile.libs ncl_ncarg-5.2.0/ni/src/ncl/yMakefile
--- ncl_ncarg-5.2.0/ni/src/ncl/yMakefile.libs	2010-03-15 20:01:01.000000000 -0600
+++ ncl_ncarg-5.2.0/ni/src/ncl/yMakefile	2010-04-28 13:56:25.401421261 -0600
@@ -121,7 +121,7 @@ DODS_LIBS	= -lnc-dap -ldap -ldapclient -
 #elif defined(Darwin)
 DODS_LIBS	= -lnc-dap -ldap -ldapclient -lcurl -lxml2 -lpthread -lssl
 #elif defined(linux)
-DODS_LIBS	= -lnc-dap -ldap -ldapclient -lcurl -lrx -lxml2 -lpthread -lidn -ldl
+DODS_LIBS	= -lcurl -lrx -lpthread
 #else
 DODS_LIBS	= -lnc-dap -ldap -ldapclient -lcurl -lrx -lxml2 -lpthread -lssl
 #endif
