--- ncl_ncarg-5.0.0/ni/src/ncl/Symbol.h.flex	2007-07-13 11:12:28.000000000 -0600
+++ ncl_ncarg-5.0.0/ni/src/ncl/Symbol.h	2007-12-13 12:32:10.000000000 -0700
@@ -523,7 +523,7 @@
 
 #ifndef YY_INPUT
 #define YY_INPUT(buf,result,max_size) \
-        if ( yy_current_buffer->yy_is_interactive ) \
+        if ( YY_CURRENT_BUFFER_LVALUE->yy_is_interactive ) \
                 { \
                 int c = '*', n; \
                 for ( n = 0; n < max_size && \
