diff -up ncl_ncarg-5.0.0/ni/src/scripts/nhlf77.csh.netcdff ncl_ncarg-5.0.0/ni/src/scripts/nhlf77.csh
--- ncl_ncarg-5.0.0/ni/src/scripts/nhlf77.csh.netcdff	2008-02-01 23:55:21.000000000 +0100
+++ ncl_ncarg-5.0.0/ni/src/scripts/nhlf77.csh	2008-02-01 23:57:23.000000000 +0100
@@ -64,7 +64,7 @@ foreach arg ($argv)
 
   case "-netcdf":
   case "-cdf":
-    set extra_libs = "$extra_libs SED_NCDFLIBS"
+    set extra_libs = "$extra_libs SED_NCDFLIBS -lnetcdff"
     breaksw
 
   case "-hdf":
diff -up ncl_ncarg-5.0.0/ni/src/scripts/ng4ex.csh.netcdff ncl_ncarg-5.0.0/ni/src/scripts/ng4ex.csh
